code/datums/gamemodes/crash.dm

```diff
@@ -145,8 +145,9 @@
 	shuttle_landed = TRUE
 
 	// We delay this a little because the shuttle takes some time to land, and we want to the xenos to know the position of the marines.
-	addtimer(CALLBACK(src, .proc/announce_bioscans, TRUE, 0, FALSE, TRUE), 30 SECONDS)  // Announce exact information to the xenos.
-	addtimer(CALLBACK(src, .proc/announce_bioscans, TRUE, 0, FALSE, TRUE), 5 MINUTES, TIMER_LOOP)
+	addtimer(CALLBACK(src, .proc/announce_bioscans, TRUE, 0, FALSE, TRUE, FALSE), 30 SECONDS)  // Announce exact information to the xenos.
+	addtimer(CALLBACK(src, .proc/announce_bioscans, TRUE, 0, FALSE, TRUE, FALSE), 5 MINUTES, TIMER_LOOP) // Xeno Regular, 5mins
```

Comment:
I've started removing procs from `TIMER_LOOP`, as I was told by ninjanomnom it wasn't a good idea to use it unless as a last resource.
Could you add this to the processing loop?

Reply:
announcement subsystem doesn't sound unreasonable.

Reply:
Wouldn't be used much, but yea, it'd be fair.

Reply:
Hard to invision how an annoucement subsystem could work, so i went with process for now.